documentation/service_admin/proc-deploying-auth-services.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-manual-steps.adoc
+
+include::common/attributes.adoc[]
+
+:cmdcli: oc
+
+[id='deploying-auth-services_{context}']
+= Deploying authentication services
+
+{ProductName} requires at least one authentication service to be deployed. The authentication service
+can either be none (allow all), standard (keycloak), or external (not managed by {ProductName}).
+
+== Deploying the none authentication service
+
+.Procedure
+
+. Create a certificate to use with the none authentication service. For testing purposes, you can create a self-signed certificate:
++
+[options="nowrap"]
+----
+mkdir -p none-authservice-cert
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=none-authservice.enmasse.svc.cluster.local" -out none-authservice-cert/tls.crt -keyout none-authservice-cert/tls.key
+----
+
+. Create a secret with the none authentication service certificate:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create secret tls none-authservice-cert --cert=none-authservice-cert/tls.crt --key=none-authservice-cert/tls.key
+----
+
+. Create the none authentication service:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create -f ./resources/none-authservice/service.yaml
+{cmdcli} create -f ./resources/none-authservice/deployment.yaml
+----
+
+== Deploying the standard authentication service
+
+.Procedure
+
+. Create a certificate to use with the standard authentication service. For testing purposes, you can create a self-signed certificate:
++
+[options="nowrap"]
+----
+mkdir -p standard-authservice-cert
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=standard-authservice.enmasse.svc.cluster.local" -out standard-authservice-cert/tls.crt -keyout standard-authservice-cert/tls.key
+----
+
+. Create a secret with the standard authentication service certificate:
```

Comment:
standard is a literal

so is secret??

Reply:
Don't think secret is a literal. It's like a password